### DISCUSSION TITLE
Convert STEP data to SheetMetal

### DISCUSSION THREAD

**Discussion by aspect-ex** [shaise/FreeCAD_SheetMetal#308] [General]
Is it possible to make an automatic BendRelief on the corner of the solid model as attached?
From image A to image B
I think if you can do this, you can convert STEP data created in other CAD to SheetMetal data in FreeCAD.

![A](https://github.com/shaise/FreeCAD_SheetMetal/assets/1826512/0003ebc5-e992-4124-b921-21bb8050f5b4)
![B](https://github.com/shaise/FreeCAD_SheetMetal/assets/1826512/4e445676-fc6c-49f0-bec9-6a15c215837e)

**Comment by shaise** [comment]
No, this is not possible. only folds made by the sheetmetal workbench can add a relief.
